src/styles/blocks/_form.scss

```diff
@@ -0,0 +1,42 @@
+.form {
```

Comment:
As far as I see, `form` classname is not used at all, so this file is not needed anymore